lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -850,6 +850,10 @@ class MongoClientInterface {
                     { error: err.message });
                 return cb(errors.InternalError);
             }
+            if (isUserBucket(c.s.name) && result.value) {
```

Comment:
Not sure I get much context around this but before you access a property's property, please make sure the parent one is defined. Otherwise it will throw an exception.